packages/sign-client/src/controllers/engine.ts

```diff
@@ -156,6 +157,8 @@ export class Engine extends IEngine {
 
     const publicKey = await this.client.core.crypto.generateKeyPair();
 
+    const expiry = ENGINE_RPC_OPTS.wc_sessionPropose.req.ttl || FIVE_MINUTES;
```

Comment:
`ENGINE_RPC_OPTS.wc_sessionPropose.req.ttl` is already a constant right, why do we need the `||` defaulting?

Reply:
it's type is actually `number | undefined` causing ts to complain

Reply:
might be better just updating the type